### PR TITLE
[Method References 2] LPS-169905 Convert method reference to ServiceTracker in class AccountRoleRequestHelper

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:account

**Comment by Kvale1733** [comment]
@W2jx No regressions.

**Comment by W2jx** [comment]
no conflicts(1/30 13:05)

**Comment by W2jx** [comment]
no conflicts(1/31 16:17)

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
ci:test:account

**Comment by W2jx** [comment]
no conflicts(2/1 10:06)

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by W2jx** [comment]
no conflicts(2/2 8:31)

File: modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/portlet/RolesAdminPortlet.java [hunk 1]
```diff
@@ -497,26 +491,6 @@ public void updateActions(
...
-				"product.navigation.personal.menu.group");
-
-		Comparator<ServiceReference<PersonalMenuEntry>> entryOrderComparator =
-			new PropertyServiceReferenceComparator<>(
-				"product.navigation.personal.menu.entry.order");
-
-		_personalMenuEntryServiceTrackerList = ServiceTrackerListFactory.open(
-			bundleContext, PersonalMenuEntry.class,
-			Collections.reverseOrder(
-				groupComparator.thenComparing(entryOrderComparator)));
```
Comment (tinatian) [inline modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/portlet/RolesAdminPortlet.java:513]: @W2jx I do not think your code follows the group comparator here

**Comment by tinatian** [comment]
@W2jx Please resend.

@W2jx Please separate your pull into two, as the first part and the second part are for different stuff, they do not have to be in the same pull.

Thanks.
Tina.

**Pull request closed without merging**